inspirehep/modules/authors/static/scss/authors/authors-profile.scss

```diff
@@ -0,0 +1,243 @@
+/**
+ *
+ * This file is part of INSPIRE.
+ * Copyright (C) 2015, 2016 CERN.
+
+ * INSPIRE is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * Invenio is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Invenio; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+// Colours.
+$background: #EDF3F4;
```

Comment:
And you can add the author profile `scss` file to https://github.com/inspirehep/inspire-next/blob/invenio3/inspirehep/modules/theme/static/scss/inspirehep.scss